components/conversations/CommentForm.js

```diff
@@ -153,6 +154,7 @@ const CommentForm = ({
           <LoadingPlaceholder height={232} />
         ) : (
           <RichTextEditor
+            defaultValue={clickedComment.id && `<blockqoute>${clickedComment.html}</blockqoute>`}
```

Comment:
Hey @Betree
Not sure why the default value is not working. Any suggestions on this?

Reply:
It looks like `RichTextEditor` only uses the `defaultValue` on its first load (in the constructor). So any change on `defaultValue` later on is simply ignored. We need to change this behavior, but at the same time, we don't want to introduce any regression in other places that may be relying on this behavior.

My suggestion would therefore be to edit `RichTextEditor` like that:
- Add a `enableReinitialize` prop (boolean)
- In `componentDidUpdate`, if `enableReinitialize` is true and the `defaultValue` has changed, reset the editor content with the default value

Trix (the underlying editor library that we use) can be confusing to work with some times, don't hesitate to ask for help if things are not straightforward.

Reply:
I have found a way, without making changes to the RichTextEditor. Using a key property on the Parent so that the child is updated when the key changes.